### PR TITLE
Landing page and new operator overview page

### LINKED ISSUE #214: New documentation landing page
As a user visiting the landing page, I want to get a quick overview of the documentation content and quickly navigate to the information I want.

Currently, the top menu is basically the only entrypoint into the content.

The introductory text currently at the top of the landing page is probably more suited for a concepts/background/explanation page, or even for stackable.tech.

## Current Idea

Some of the documentation I like uses tiles (see examples below, with tiles we can show people what we think is important, and write a sentence of description on the tile as well, to help first time users quickly find what they need.

image

### PR DESCRIPTION
**Pull request by fhennig**
New landing page to help users find better what they are looking for.

- The landing page was updated
- The operator overview page updated
- The tutorial page updated

The CRD docs are now linked too.

fixes #214, #455

### CODE REVIEW

**Review by NickLarsenNZ — DISMISSED** [review]
This looks way nicer!

**Comment by fhennig** [comment]
Thank you!

**Comment by fhennig** [comment]
TODO: Link release notes from landing page

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -148,3 +148,9 @@ Learn more about the tooling that supports the Stackable Data Platform; the `sta
 ++++
 <div id="help-action">Still need help? Ask a question on <a href="https://github.com/orgs/stackabletech/discussions">GitHub</a> or <a href="https://discord.com/invite/7kZ3BNnCAF">Discord</a>.</div>
 ++++
+
+== Release Notes
+
+The Stackable platform consists of multiple operators that work together. Periodically a platform release is made,
+including all components of the platform at a specific version. See the latest release notes for 23.7
```
[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply modules/ROOT/pages/index.adoc]: Removed in https://github.com/stackabletech/documentation/pull/492/commits/7e11a4dbfae949d1910fa45f9717b3ad887f0f12

**Review by NickLarsenNZ — APPROVED** [review]
LGTM

**Pull request merged**